zcash_client_backend/src/data_api.rs

```diff
@@ -338,6 +341,10 @@ impl<Nf> ScannedBlock<Nf> {
         &self.transactions
     }
 
+    pub fn sapling_nullifier_map(&self) -> &[(TxId, u16, Vec<sapling::Nullifier>)] {
```

Comment:
Again please add documentation (it can be in one place and referred to from the other place).

Reply:
The entire struct needed documenting, so we can do this before the next release.

Reply:
Created https://github.com/zcash/librustzcash/issues/885